docs/pipeline-components-and-applications/snowplow-micro/index.md

```diff
@@ -0,0 +1,545 @@
+      "br_features_director":null,
+      "br_features_quicktime":null,
+      "br_features_realplayer":null,
+      "br_features_windowsmedia":null,
+      "br_features_gears":null,
+      "br_features_silverlight":null,
+      "br_cookies":true,
+      "br_colordepth":"24",
+      "br_viewwidth":1000,
+      "br_viewheight":660,
+      "os_name":null,
+      "os_family":null,
+      "os_manufacturer":null,
+      "os_timezone":"Europe/London",
+      "dvce_type":null,
+      "dvce_ismobile":null,
+      "dvce_screenwidth":1920,
+      "dvce_screenheight":1080,
+      "doc_charset":"UTF-8",
+      "doc_width":988,
+      "doc_height":670,
+      "tr_currency":null,
+      "tr_total_base":null,
+      "tr_tax_base":null,
+      "tr_shipping_base":null,
+      "ti_currency":null,
+      "ti_price_base":null,
+      "base_currency":null,
+      "geo_timezone":null,
+      "mkt_clickid":null,
+      "mkt_network":null,
+      "etl_tags":null,
+      "dvce_sent_tstamp":"2021-08-29T12:01:05.752Z",
+      "refr_domain_userid":null,
+      "refr_dvce_tstamp":null,
+      "derived_contexts":{},
+      "domain_sessionid":"6d15a4fb-9623-4ba1-b876-5240e72e6970",
+      "derived_tstamp":"2021-08-29T12:01:05.783Z",
+      "event_vendor":"com.snowplowanalytics.snowplow",
+      "event_name":"page_view",
+      "event_format":"jsonschema",
+      "event_version":"1-0-0",
+      "event_fingerprint":null,
+      "true_tstamp":null
+    }
+  }
+]
+```
+
+#### Filters
+
+When querying `/micro/good` with `POST` (`Content-Type: application/json` needs to be set in the headers of the request), it's possible to specify filters, thanks to a JSON in the data of the HTTP request.
+
+Example of command to query the good events: 
+
+```bash
+curl -X POST -H 'Content-Type: application/json' <IP:PORT>/micro/good -d '<JSON>'
+```
+
+An example of JSON with filters could be:
+
+```json
+{ 
+  "schema": "iglu:com.acme/example/jsonschema/1-0-0",
+  "contexts": [
+    "com.snowplowanalytics.mobile/application/jsonschema/1-0-0",
+    "com.snowplowanalytics.mobile/screen/jsonschema/1-0-0"
+  ],
+  "limit": 10
+}
+```
+
+List of possible fields for the filters:
+
+- `event_type`: type of the event (in `e` param);
+- `schema`: corresponds to the shema of an [unstructured event](/docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md#Custom_unstructured_event_tracking) (schema of the self-describing JSON contained in `ue_pr` or `ue_px`). It automatically implies `event_type` = `ue`.
+- `contexts`: list of the schemas contained in the contexts of an event (parameters `co` or `cx`). An event must contain **all** the contexts of the list to be returned. It can also contain more contexts than the ones specified in the request.
+- `limit`: limit the number of events in the response (most recent events are returned).
+
+It's not necessary to specify all the fields in a request, only the ones that need to be used for filtering.
```

Comment:
If we're going to keep current format, could we make the main headers level 2 and push the filter headers to a level 3 so they show obviuously in the sidebar?